jenkins-integration/beaker/install/foss/40_clone_test_catalogs.rb

```diff
@@ -0,0 +1,16 @@
+test_name 'Clone test-catalogs repo onto master'
+
+# This is awful - we clone locally and then SCP the whole thing over.
+# See QENG-2556 for adding a proper way to do this in Beaker.
+tmp_repo = "/tmp/test-catalogs-#{Process.pid}"
+target = '/root/test-catalogs'
+
+teardown do
+  %x(rm -rf #{tmp_repo})
```

Comment:
Yeah, I dunno.  I guess `File.directory?` by itself wouldn't be any safer.  Just trying to think of safeguards to prevent a `rm -rf /`.  It's not a huge deal since these are just test machines, but if this were production code I'd want to put in *some* kind of safeguard.  Anyway, I'm fine with it as-is for now.